recipes-browser/wpewebkit/wpewebkit_2.40.0.bb

```diff
@@ -0,0 +1,39 @@
+require wpewebkit.inc
+require conf/include/devupstream.inc
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+
+SRC_URI = "https://wpewebkit.org/releases/${BPN}-${PV}.tar.xz;name=tarball \
+           file://0001-FELightningNEON.cpp-fails-to-build-NEON-fast-path-se.patch \
+          "
+
+SRC_URI[tarball.sha256sum] = "a8ec2bcfa1613768ab3ce7f65cac4f214835266c2ff59440d72180baec3086b6"
+
+DEPENDS += " libwpe"
+RCONFLICTS:${PN} = "libwpe (< 1.12)"
```

Comment:
Check (and update if it is required) the dependencies in https://wpewebkit.org/release/schedule/

Reply:
... and the same for the rest of the components (libwpe, wpebackend-fdo ...)